### PR TITLE
feat(mod): Add mod function (#86)

### LINKED ISSUE #86: Add modulo function that only returns positive values.
Hi, I would like to add a "mod" function that does modulo but only returns positive values if you supply a negative dividend. Here is the [stack overflow solution](http://stackoverflow.com/a/17323608) I would like to use.

### PR DESCRIPTION
**Pull request by tbarber350**
Hi, I believe this is correct. If anything looks wrong just let me know. Thanks!
Closes #86

### CODE REVIEW

File: src/mod.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export default mod
+
+/**
+ * Original Source: http://stackoverflow.com/a/17323608
+ *
+ * This method provides a modulo function that will always return a positive
+ * value even when supplied a negative dividend.
+ *
+ * @param {Date} date - date to be evaluated
+ * @returns {Boolean} - True if date is today, false otherwise
```
Comment (kentcdodds) [inline src/mod.js]: I think this JSDoc is describing the params/returns incorrectly. Looks like copy/paste 😅

File: test/mod.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const expected = 4
+  const result = mod(dividend, divisor)
+  t.is(expected, result)
+})
+
+test('a negative dividend', t => {
+  const dividend = -12
+  const divisor = 8
+  const expected = 4
+  const result = mod(dividend, divisor)
```
Comment (kentcdodds) [inline test/mod.test.js]: Don't forget to add an assertion (`t.is`) :)

Comment (tbarber350) [inline test/mod.test.js]: Oh man that was sloppy. Sorry about that. I thought I had swapped out that documentation.  I have made those updates. Thanks!

**Comment by kentcdodds** [comment]
Great job! Thanks for the contribution :tada:

**Comment by tbarber350** [comment]
Thanks! The egghead course is great! Thanks for putting it together.

**Pull request merged**